client/src/pages/donate.tsx

```diff
@@ -60,8 +61,9 @@ function DonatePage({
       action: `Displayed Donate Page`
     });
   }, []);
+  const growthbook = useGrowthBook();
 
-  return showLoading ? (
+  return showLoading || !growthbook || !growthbook.ready ? (
```

Comment:
Do we need this? What happens if growthbook is down?

Reply:
If growthbook is down, the request will fail and the detfault features will load. 

We can take this part out.